src/test/utils/getMembers.test.js

```diff
@@ -18,39 +18,51 @@ import {
 
 // getEveryMember(array, fn) = array.filter(fn)
 test('getEveryMember()', () => {
-  expect(getEveryMember([ 0, 1, 0 ], (val) => {return val === 0;})).toHaveLength(2);
+  expect(getEveryMember([ 0, 1, 0 ], (val) => {
+    return val === 0;}
```

Comment:
Curly brace on next line.

Reply:
Yeah, this is the one that needs to be changed to finish with the linting stuff.